cl_sii/rtc/data_models.py

```diff
@@ -245,9 +275,9 @@ def validate_dte_tipo_dte(cls, v: object) -> object:
         return v
 
     @pydantic.validator('fecha_cesion_dt')
-    def validate_datetime_tz(cls, v: object) -> object:
+    def validate_fecha_cesion_dt(cls, v: object) -> object:
         if isinstance(v, datetime):
-            tz_utils.validate_dt_tz(v, cls.DATETIME_FIELDS_TZ)
```

Comment:
Does it make sense to nesting `tz_utils.validate_dt_tz()` into `validate_cesion_fecha()`?